Data/HashMap/Strict/Base.hs

```diff
@@ -615,6 +622,12 @@ fromListWith :: (Eq k, Hashable k) => (v -> v -> v) -> [(k, v)] -> HashMap k v
 fromListWith f = L.foldl' (\ m (k, v) -> unsafeInsertWith f k v m) empty
 {-# INLINE fromListWith #-}
 
+-- | /O(n*log n)/ Construct a map from a list of elements.  Uses
+-- the provided function f to merge duplicate entries (f key newVal oldVal).
```

Comment:
Docs here could use a bit of help. Suggesting a patch: 

```suggestion
-- the provided function @f@ to merge duplicate entries @(f key newVal oldVal)@.
```

Reply:
Done, thanks!